spec/Dockerfile.puppetserver

```diff
@@ -1,8 +1,9 @@
-FROM puppet/puppetserver-standalone
+FROM puppet/puppetserver
 
 ARG hostname="boltserver"
 ENV PUPPETSERVER_HOSTNAME "$hostname"
-
+ENV PUPPET_STORCONFIGS false
```

Comment:
```suggestion
ENV PUPPET_STORECONFIGS false
```

Reply:
It actually doesn't look like puppetserver honors either of these settings? https://puppet.com/docs/puppetserver/latest/configuration.html

Reply:
The page is kind of misleading. I think it refers to the settings which affect the behavior of the clojure code in puppetserver. But puppetserver also runs puppet in jruby instances and those *do* honor the rest of puppet.conf.

Reply:
Updated with the correct env var name.

Reply:
oic on the pr. :+1: